apps/src/templates/textResponses/TextResponses.jsx

```diff
@@ -52,9 +149,29 @@ class TextResponses extends Component {
             onChange={this.onChangeScript}
           />
         </div>
+        {filteredResponses.length > 0 &&
+          <div
+            id="uitest-response-actions"
+            style={styles.actionRow}
+          >
+            <div>{this.renderFilterByStageDropdown()}</div>
+            <CSVLink
+              style={styles.buttonContainer}
+              filename="responses.csv"
+              data={filteredResponses}
+              headers={CSV_HEADERS}
+            >
+              <Button
+                text={i18n.downloadCSV()}
+                onClick={() => {}}
```

Comment:
nothing happens when this button is clicked?

Reply:
you could use `href="#download"` instead? and add a comment saying the click event is handled by `CSVLink`?

Reply:
`CSVLink` renders as an anchor tag, so `Button` having an `href` causes an error because an anchor tag cannot be a child of another anchor tag 😢 

i added a comment above `Button` explaining that the functionality for `onClick` is handled by `CSVLink`. another thing i could do (if we think it's necessary) is build my own `Button` component that uses the same styles but doesn't require `onClick`

Reply:
...and i pushed it to my next branch for this feature (`text-responses-react-pt3`) 🤦‍♀️ so i'll merge this and continue tweaking feedback